pythonsdk/src/main/python/vantiqservicesdk.py

```diff
@@ -177,13 +184,28 @@ async def __process_message(self, websocket: WebSocket, msg_bytes: bytes) -> Non
                 result = await self.__invoke(procedure_name, params)
                 response["result"] = result
 
+            # Serialize to JSON and encode
+            text = json.dumps(response, separators=(",", ":"), ensure_ascii=False,
```

Comment:
Need to do this under the `try` so that when there are serialization errors, the client sees them.